setup.md

```diff
@@ -0,0 +1,23 @@
+Initial setup
```

Comment:
Headers in Markdown are

```
# Initial Setup
```

Can we put the contents of `setup.md` into `README.md` under a `## Setup` heading? I think it is best to have all of this in one location